guides/v2.2/magento-functional-testing-framework/release-2/test.md

```diff
@@ -127,15 +133,34 @@ Attribute|Type|Use|Description
 `before`|string|optional| `<stepKey>` of an action or action group that must be executed next while merging.
 `after`|string|optional| `<stepKey>` of an action or action group that must be executed one step before the current one while merging.
 
-`<actionGroup>` may contain [`<argument>`](#argument-tag).
+`<actionGroup>` may contain [`<argument>`].
 
 ### argument {#argument-tag}
 
-`<argument>` sets an argument that is used in the parent [`<actionGroup>`](#actiongroup-tag).
+`<argument>` sets an argument that is used in the parent [`<actionGroup>`].
 
 Attribute|Type|Use
 ---|---|---
 `name`|string|optional| Name of the argument.
 `value`|string|optional| Value of the argument.
 
-See more about [Action groups](./test/action-groups.html).
+See more about [Action groups][action group].
```

Comment:
See [Action groups][action group] for more information.

Reply:
🆗